addons/purchase/wizard/bill_to_po_wizard.py

```diff
@@ -0,0 +1,82 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models, Command, _
+from odoo.exceptions import UserError
+
+
+class BillToPO(models.TransientModel):
+    _name = 'bill.to.po.wizard'
+    _description = 'Bill to Purchase Order'
+
+    display_add_button = fields.Boolean()
+    purchase_order_id = fields.Many2one(comodel_name='purchase.order')
+    partner_id = fields.Many2one(comodel_name='res.partner')
+
+    # @api.depends_context('active_ids')
+    # def _compute_aml_ids(self):
+    #     print('_compute_aml_ids')
+    #     aml_ids = [abs(record_id) for record_id in self.env.context.get('active_ids') if record_id < 0]
+    #     for wizard in self:
+    #         wizard.aml_ids = self.env['account.move.line'].browse(aml_ids)
```

Comment:
Remove:
this was an attempt to remove the duplicated line:
`aml_ids = [abs(record_id) for record_id in self.env.context.get('active_ids') if record_id < 0]`
it didn't work well.